src/privileged/notificationBar/api.js

```diff
@@ -0,0 +1,155 @@
+"use strict";
+
+/* global ExtensionAPI */
+
+ChromeUtils.import("resource://gre/modules/Console.jsm");
+ChromeUtils.import("resource://gre/modules/Services.jsm");
+ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionUtils.jsm");
+
+/* eslint-disable-next-line */
+var {EventManager, EventEmitter} = ExtensionCommon;
+/* eslint-disable-next-line no-var */
+var {Management: {global: {tabTracker}}} = ChromeUtils.import("resource://gre/modules/Extension.jsm", {});
```

Comment:
Interesting that you're importing another API here like this. Is this a stable approach?

Reply:
Probably not, still works in 63 though (and I don't see them uplifting code that changes this), and I haven't found a better way to do it.